lib/generator.js

```diff
@@ -86,7 +86,31 @@ const wrapType = (type, customTypes = [], usedTypes) => {
   return type;
 };
 
+const splitEscaped = (line, sep = ' ') => {
+  const result = [];
+  let i = 0;
+  let start = line.indexOf('`', i);
+  while (i < line.length && start !== -1) {
+    result.push(...line.slice(i, start).split(sep));
```

Comment:
This doesn't seem to handle preceeding symbols before the `` ` `` at the start of the line (i.e.
`` aaa78symbolsline`abc` ``), does it?

Reply:
It does. I've updated example as well to show this.

Reply:
looking at this line `result.push(...line.slice(i, start).split(sep));` if we consider the `line` ===  ``Start`abc` continue`` this will call `slice(0, 5).split(' ')` which will result in `['Start']` therefore separating it from the backticks, wouldn't it? Or it this handled somewhere else?

Reply:
If you look here
https://github.com/metarhia/metadoc/blob/2737d0dffda55d39fa2facea091673ac0e2a50a2/lib/generator.js#L104
you will see that escaped text will always be appended to the previous line: `Start` in this case and empty line in case if there is a separator or beginning of the line before escaped text.